genesyscloud/dependent_consumers/dependent_consumers.go

```diff
@@ -0,0 +1,45 @@
+package dependent_consumers
+
+var dependentConsumerMap map[string]string
+
+func SetDependentObjectMaps() map[string]string {
+	if len(dependentConsumerMap) < 1 {
+		dependentConsumerMap = make(map[string]string)
```

Comment:
I compared the types that are returned via the API `/api/v2/architect/dependencytracking/types` and it looks like you're missing the following mappings:
```
dependentConsumerMap["COMPOSERSCRIPT"] = "genesyscloud_script"
dependentConsumerMap["ACDLANGUAGE"] = "genesyscloud_routing_language"
dependentConsumerMap["INQUEUECALLFLOW"] = "genesyscloud_flow"

```

You should also research whether mappings need to be added for these other dependency tracking types:
- botFlow
- commonModuleFlow
- dataAction
- flowAction
- secureAction
- secureCallFlow
- surveyInviteFlow
- voicemailFlow
- workflow

Reply:
Also, please sort this list after you have finished confirming that this list is complete

Reply:
Hi @bbbco and @HemanthDogiparthi12 

We definitely want data actions and the other items above.  What is interesting is that the botFlow, commonModuleFlow, secureCallFlow, surveyInviteFlow, voicemailFlow, and workflow all map back to the flow resource.  Which raises a good point?  Have should probably do some testing to make sure that when there is a dependency on another flow we are properly exporting all of the dependencies

Reply:
Added the resources mentioned